webapp/src/components/BuyParcelByMortgagePage/BuyParcelByMortgagePage.js

```diff
@@ -80,7 +80,11 @@ export default class BuyParcelByMortgagePage extends React.PureComponent {
       return this.renderNotConnected()
     }
 
-    const { isMortgageApprovedForMana, isMortgageApprovedForRCN } = wallet
+    const {
+      isMortgageApprovedForMana,
+      isMortgageApprovedForRCN,
```

Comment:
i think this doesn't work with the current authorizations model

cc @NicoSantangelo

Reply:
Yup! it was bound to be a conflict post merge :(